scripts/install_ddev_wsl2_docker_desktop.ps1

```diff
@@ -41,8 +34,41 @@ if (-not(wsl -e docker ps) ) {
     throw "Docker Desktop integration with the default distro does not seem to be enabled yet."
 }
 $ErrorActionPreference = "Stop"
-# Install needed choco items
-choco upgrade -y ddev gsudo mkcert
+
+# Install DDEV on Windows to manipulate the local hosts file.  (Also requires mkcert & sudo; see below.)
+$TempDir = $env:TEMP
+$DdevInstallerPath = Join-Path $TempDir "ddev-installer.exe"
```

Comment:
This is just the right approach. I had been thinking of not using the installer, but using the installer is great. Then you get mkcert and gsudo for free, don't have to worry about those at all. Perfect.